Models/InertDoubletModel/exampleCollisionDefs.py

```diff
@@ -148,52 +138,25 @@ def newScalarThermalMassSquared(p: WallGoCollision.ModelParameters) -> float:
 
     wBoson = WallGoCollision.ParticleDescription()
     wBoson.name = "W"
-    wBoson.index = 6 #7
+    wBoson.index = 4
     wBoson.type = WallGoCollision.EParticleType.eBoson
     wBoson.bInEquilibrium = False
     wBoson.bUltrarelativistic = True
     wBoson.massSqFunction = wBosonThermalMassSquared
     modelDefinition.defineParticleSpecies(wBoson)
 
     higgs = WallGoCollision.ParticleDescription()
-    higgs.name = "h"
-    higgs.index = 7 #8
+    higgs.name = "Higgs"
+    higgs.index = 5
     higgs.type = WallGoCollision.EParticleType.eBoson
     higgs.bInEquilibrium = True
     higgs.bUltrarelativistic = True
     higgs.massSqFunction = HiggsBosonThermalMassSquared
     modelDefinition.defineParticleSpecies(higgs)
 
-    # goldStone0 = WallGoCollision.ParticleDescription()
```

Comment:
Yes, I overlooked it. Thanks for finding it

Reply:
You mean the Higgs?

Reply:
No, actually they are part of the Higgs doublet

Reply:
Ok so I actually meant the stuff that was deleted. So there we agree that it was light? Then the Higgs should probably remain there. But the goldstones are now gone.

Reply:
At some point the Higgs doublet was split into separate representations for the Higgs (1 dof) and the Goldstones. Then in the collision defs, I listed them separately. They were not listed with the light reps.

Now (but this actually happened already before this PR, this deleted stuff was already commented out), the Higgs doublet (including Goldstones) is tracked in a single representation, and it is not listed as light. So maybe this is a bit of a misnomer, because it really denotes Higgs + Goldstones.

Reply:
perhaps should be named HiggsDoublet. Should not hold back the approval of this PR though.